src/services/apis.rs

```diff
@@ -0,0 +1,52 @@
+use serde_json::Value;
+
+use crate::{
+    models::{ListKeysRequest, ListKeysResponse},
+    routes::{self, CompiledRoute},
+    services::HttpService,
+    types::Response,
+    unwind_response,
+};
+
+#[allow(unused_imports)]
+use crate::types::HttpError;
```

Comment:
This import is only present in the `services/keys.rs` file because it is used in a documentation comment. Either use it that way, or remove this. (preferably the former).